changelog/1.17.3.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "1.17.3"
+description: "Released on 04/08/21"
+---
+
+### Breaking Changes ❗
+
+There are no breaking changes in 1.17.3.
+
+### Features ✨
+
+There are no new features in 1.17.3.
+
+### Bug Fixes 🐛
+
+- infra: Fixes an issue whereby workspace application icons appear broken and
+  the workspace provider ping indicator always displays as an error. The
+  underlying cause was a malformed _envproxy_access_url_ on `built-in` workspace
+  providers
```

Comment:
@khorne3 

Since the workspace provider ping indicator is somewhat of a UI element, I'm wondering if it warrants bold fontface

![Screenshot from 2021-04-08 12-53-33](https://user-images.githubusercontent.com/11184711/114073980-80832b00-9869-11eb-8153-bfd66f0fb5e2.png)

I imagine we can also reference the **Applications** UI element

![Screenshot from 2021-04-08 12-54-15](https://user-images.githubusercontent.com/11184711/114074040-9133a100-9869-11eb-97eb-e58f3bed79d3.png)

Reply:
Yeah, I would put "ping indicator" in bold for emphasis